internal/telemetry/telemetry.go

```diff
@@ -75,3 +76,33 @@ func WrapHandler(handler http.Handler) http.HandlerFunc {
 		span.SetAttributes(semconv.HTTPResponseContentLengthKey.Int64(m.Written))
 	}
 }
+
+type SlogHandler struct {
+	handler slog.Handler
+}
+
+func NewSlogHandler(h slog.Handler) slog.Handler {
+	return &SlogHandler{handler: h}
+}
+
+func (h *SlogHandler) Enabled(ctx context.Context, level slog.Level) bool {
+	return h.handler.Enabled(ctx, level)
+}
```

Comment:
Is this function and `WithGroup()` here to satisfy an interface as they don't appear to be doing anything or be in use?